ui/src/components/configuration/sdk-config-docs/cordova.js

```diff
@@ -0,0 +1,107 @@
+const framework = docsVersion => ({
+  icon: '/img/cordova.png',
+  title: 'Cordova',
+  steps: [
+    {
+      introduction:
+        'First step is to install the Core AeroGear module, the module manages and binds all services together on the client side.',
+      commands: [
+        'Open a terminal and navigate to your appplication project root folder.',
+        ['Install AeroGear Core package, from terminal execute', '```npm install @aerogear/app```'],
+        [
+          'In your application, import and call the "init" method:',
+          `
+        import { init } from "@aerogear/app";
+
+        const aerogearConfig = {
+            // Replace with your own configuration
+        };
+        
+        init(aerogearConfig);`
+        ]
+      ]
+    },
+    {
+      introduction:
+        'Copy `mobile-services.json` content and copy it to the location `src/mobile-services.json` in your application project.'
+    }
```

Comment:
I think the instructions here are not accurate. Especially around when to copy the `mobile-services.json` file and how to init. The right order should be (after installing the core package):

1. Copy `mobile-services.json` file to the app's source folder.
2. To init, load the content from `mobile-services.json` file, and pass it to init:
    ```
    import { init } from "@aerogear/app";
    const appConfig = require("./mobile-services.json");
    init(appConfig);
    ```